motionplan/collision.go

```diff
@@ -1,12 +1,16 @@
 package motionplan
 
 import (
+	"fmt"
 	"math"
+	"strconv"
 
 	spatial "go.viam.com/rdk/spatialmath"
 	"go.viam.com/rdk/utils"
 )
 
+const unnamedCollisionGeometryPrefix = "unnamedCollisionGeometry_"
```

Comment:
This is entirely internal so the name doesn't really matter, unless there is cost associated with hashing longer names.  Is this not an issue?